### PR TITLE
Changes regarding ezplatform/2.0

### PR DESCRIPTION
**Pull request by SylvainGuittard**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -53,26 +54,26 @@ https://doc.ez.no/display/DEVELOPER/Step+1%3A+Installation
 
        ```bash
        $ curl -sS https://getcomposer.org/installer | php
-       $ php -d memory_limit=-1 composer.phar create-project --no-dev ezsystems/ezplatform
+       $ php -d memory_limit=-1 composer.phar create-project ezsystems/ezplatform ezplatform ^2.0
```
Comment (andrerom) [inline INSTALL.md]: `ezsystems/ezplatform` => `ezsystems/ezplatform-demo`

File: app/config/ezplatform.yml [hunk 2]
```diff
@@ -57,10 +57,6 @@ ezpublish:
             # HttpCache purge server(s) setting, eg Varnish, for when ezpublish.http_cache.purge_type is set to 'http'.
             http_cache:
                 purge_servers: ['%purge_server%']
-            # Temporary hard coding session name during v2 development
-            session:
-                name: eZSESSID
-            # ez-design-engine configuration
```
Comment (andrerom) [inline app/config/ezplatform.yml]: should also drop `session` from admin_group

Comment (SylvainGuittard) [reply app/config/ezplatform.yml]: Thanks. Updated in #70

File: INSTALL.md (hunk 1 above)
Comment (SylvainGuittard) [reply INSTALL.md]: Thanks @andrerom 
I think it makes more sense to keep the current install steps on the README in order to not confuse people. So I submitted a new PR without INSTALL.MD

**Comment by SylvainGuittard** [comment]
Closing in favor of #70

**Pull request closed without merging**